.github/workflows/extended-checks.yml

```
@@ -51,8 +51,10 @@ jobs:
         run: |
           ./start &
           sleep 20
+        if: always()
       - name: Check current API pages render
         run: >
           npm run check-pages-render --
           --qiskit-release-notes
           --current-apis
+        if: always()
```

```suggestion
        if: !cancelled()
      - name: Check current API pages render
        run: >
          npm run check-pages-render --
          --qiskit-release-notes
          --current-apis
        if: !cancelled()
```

Nit as we probably won't encounter this issue, but `always` will run even if the job is cancelled.